xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecAndroidMediaCodec.cpp

```diff
@@ -749,13 +750,45 @@ bool CDVDVideoCodecAndroidMediaCodec::Open(CDVDStreamInfo &hints, CDVDCodecOptio
 
     CLog::Log(LOGINFO, "CDVDVideoCodecAndroidMediaCodec::Open Testing codec: {}", m_codecname);
 
+    // There is some confusion for video/VC1 vs. video/wvc1 especially on Sony devices, while
+    // IANA defines VC-1 as video/vc1, Android API defines it as video/wvc1 - be nice and test it
+    bool isVC1 = StringUtils::Contains(m_codecname, "vc1", false);
+
     CJNIMediaCodecInfoCodecCapabilities codec_caps = codec_info.getCapabilitiesForType(m_mime);
     if (xbmc_jnienv()->ExceptionCheck())
     {
       // Unsupported type?
       xbmc_jnienv()->ExceptionDescribe();
       xbmc_jnienv()->ExceptionClear();
-      continue;
+      if (isVC1)
+      {
+        const std::vector<std::string> mimes = { "video/VC1", "video/vc1" };
```

Comment:
Better is use `std::array` here and allocate in stack (is not need capability of push / remove elements)

Reply:
I chose the vector, cause it was already included. But fine with me. Thanks.